style.css

```diff
@@ -0,0 +1,273 @@
+* {
+    font-family: sans-serif;
+}
+body {
+  margin: 0;
+}
+.grid {
+  grid-template-columns: 300px 1fr 400px;
+  grid-template-rows: 100vh;
+}
+.left-bar,
+.right-bar,
+header,
+.text,
+.shadow-gap,
+.content {
+  border: solid 1px rgb(172, 172, 172);
+}
+.g, .grid {
+  display: grid;
+}
+.shadow-gap {
+    background-color: rgb(208, 208, 208);
+}
+.prof-border {
+  margin-left: 5px;
+  margin-bottom: 15px;
+  border: solid 1px black;
+  width: 40px;
+  height: 40px;
+  border-radius: 50%;
+  overflow: hidden;
+}
+.profile-pic {
+  width: 100px;
+  position: relative;
+  bottom: 17px;
+  right: 30px;
+}
+.legollas {
+    width: 70px;
+    bottom: 25px;
+    right: 20px;
+}
+.legolas-image {
+    width: 80%;
+}
+.gandalf {
+    right: 41px;
+    margin-bottom: 10px;
+}
+#refresh {
+    justify-content: end;
+    padding-right: 5px;
+    padding-top: 5px;
+}
+.left-bar {
+  justify-content: end;
+  grid-template-rows: 1fr 8fr 1fr 2fr;
+}
+.left-bar > div:nth-child(1) > img, .prof-border {
+  padding-top: 10px;
+  padding-left: 10px;
+}
+.logo img {
+  position: relative;
+  top: 7px;
+  height: 30px;
+  width: 30px;
+  margin-right: 12px;
+  padding-left: 5px;
+}
+nav button:nth-child(1) > img {
+  background-color: yellow;
+  border-radius: 50%;
+}
+nav button {
+  font-size: 20px;
+  font-weight: bold;
+}
+.tweet button {
+  width: 80%;
+  height: 50px;
+  background-image: radial-gradient(blue, black);
+  border: solid 1px black;
+  border-radius: 25px;
+  color: rgb(255, 255, 255);
+  font-size: 20px;
+  font-weight: bold;
+  margin-left: 10px;
+}
+.logo button {
+  text-align: start;
+  background-color: rgba(255, 255, 255, 0);
+  border: none;
+}
+.logo,
+.profile,
+.tweet {
+  padding-top: 5px;
+  width: 200px;
+}
+.profile {
+  align-content: end;
+  grid-template-columns: 60px 1fr 40px;
+}
+.profile p {
+  margin: 0;
+}
+.profile > div + div > p:nth-child(1) {
+  margin-top: 10px;
+}
+span {
+  font-size: 13px;
+  color: gray;
+}
+.elipses-grid {
+    grid-template-columns: 1fr 50px;
+}
+.elipses {
+  margin-top: 20px;
+  margin-left: 15px;
+  width: 13px;
+  justify-self: end;
+  padding-right: 10px;
+}
+.logoz {
+    margin-top: 20px;
+    width: 30px;
+    height: 30px;
+}
```

Comment:
Good use making classes to add repetitive effects to multiple sections of the HTML